zipline/finance/trading.py

```diff
@@ -93,13 +92,34 @@ def __init__(
         max_date=None,
         env_trading_calendar=tradingcalendar
     ):
+        # `tc_tdays` is short for "trading calendar trading days"
+        # `tc_tday` is short for "trading calendar trading day"
+        tc_tdays = env_trading_calendar.trading_days
+        tc_tday = env_trading_calendar.trading_day
```

Comment:
Ah, got it, thanks. 

The purpose of adding the tc_tday field, then, is to expose this Custom Business Day object through the TradingEnvironment's 'trading_day' field? Where would this field be read?